tools/document-pii/src/item_collector.rs

```diff
@@ -0,0 +1,251 @@
+use std::collections::{BTreeMap, BTreeSet, HashMap};
+use std::fs;
+use std::io::BufRead;
+use std::path::{Path, PathBuf};
+
+use crate::EnumOrStruct;
+
+use anyhow::anyhow;
+use syn::punctuated::Punctuated;
+use syn::visit::Visit;
+use syn::{ItemEnum, ItemStruct, UseTree};
+
+/// Iterates over the rust files to collect all the types and use_statements, which is later
+/// used for recursively looking for pii_fields afterwards.
+pub struct ItemCollector {
```

Comment:
Since we're dealing with the AST:
```suggestion
pub struct AstItemCollector {
```